test/verify/check-machines

```diff
@@ -3730,6 +3731,15 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
                 else:
                     b.wait_not_present("#create-network-dialog")
 
+            def verify_expected_error_on_head(self, error_message):
+                b.wait_present("#app > div > section > div > div")
+                # Click show more button
+                b.click("#app > div > section > div > div > h4 > button")
```

Comment:
Please avoid using selectors like this long, relying on divs and such, because every small change i do in that part of the code will need test adjustment. Better rely on classes or ids.
No need to repost for this.

Reply:
OK, I will pay more attention in the future, many thanks.